pbench-agent.html

```diff
@@ -169,8 +169,9 @@ <h2 id="sec-4"><span class="section-number-2">4</span> TL;DR version</h2>
 	      Here is the minimum set of commands to get from 0 to a crawl:
 	    </p>
 	    <pre><code>
-        wget -O /etc/yum.repos.d/pbench.repo http://repohost.example.com/repo/yum.repos.d/pbench.repo
-        yum install pbench-agent -y
+	sudo wget -O /etc/yum.repos.d/pbench.repo https://raw.githubusercontent.com/redhat-performance/openshift-performance/master/repos/pbench.repo
```

Comment:
can we maybe merge with intent to improve?  right now what is in the instructions is literally completely useless